MatrixMultiplication/MatrixMultiplicationTests/MatrixMultiplicationTests.cs

```diff
@@ -0,0 +1,46 @@
+namespace MatrixMultiplication.Tests;
+
+public class MatrixMultiplicationTests
+{
+    public static IEnumerable<TestCaseData> MultiplierAlgo()
+    {
+        yield return new TestCaseData(new NonParallelMultiplicationAlgo());
+        yield return new TestCaseData(new ParallelMultiplicationAlgo());
+    }
+
+    [TestCaseSource(nameof(MultiplierAlgo))]
+    public void Multiply1x1Matricies(IMultiplicationAlgo multiplicationAlgo)
+    {
+        int[,] left = MatrixMultiplier.InputMatrix("Samples/Test1/left.txt");
+        int[,] right = MatrixMultiplier.InputMatrix("Samples/Test1/right.txt");
```

Comment:
```suggestion
        var right = MatrixMultiplier.InputMatrix("Samples/Test1/right.txt");
```
Удобнее же не набирать каждый раз эти 6 символов со скобками и запятыми